app/services/FirebaseService.scala

```diff
@@ -1,16 +1,31 @@
 package services
 
-import com.google.firebase.database.{DatabaseReference, FirebaseDatabase}
+import com.google.firebase.database._
+import play.api.Logger
+import actor.ChatSystem
+
 import scala.collection.JavaConverters._
 
 object FirebaseService {
   val database: FirebaseDatabase = FirebaseDatabase.getInstance()
 
-  def ref: DatabaseReference = database.getReference("chat/Room")
+  def ref: DatabaseReference = database.getReference("chat")
 
   def save(name: String) = {
     val data: Map[String, AnyRef] = Map("Create" -> List(name).asJava)
     ref.updateChildrenAsync(data.asJava)
```

Comment:
Because you change Database reference from `"chat/Room"` to `"chat"` this will cause bug on `def save`